locale/fr/LC_MESSAGES/index.po

```diff
@@ -0,0 +1,160 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2021 Fortran programming language community
+# This file is distributed under the same license as the fpm package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: fpm\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-01-14 10:30+0100\n"
+"PO-Revision-Date: 2022-01-14 15:04+0100\n"
+"Last-Translator: Vincent MAGNIN <vincent.magnin@libertysurf.fr>\n"
+"Language-Team: \n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.9.1\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#: ../../pages/index.md:141
+msgid "Tutorial"
+msgstr "Tutoriels"
+
+#: ../../pages/index.md:141
+msgid "How-To"
+msgstr "Guides pratiques"
+
+#: ../../pages/index.md:141
+msgid "Design"
+msgstr "Comprendre fpm"
+
+#: ../../pages/index.md:141
+msgid "Reference"
+msgstr "Manuel de référence"
+
+#: ../../pages/index.md:141
+msgid "Registry"
+msgstr "Catalogue de paquets fpm"
+
+#: ../../pages/index.md:1 ../../pages/index.md:5
+msgid "Fortran Package Manager"
+msgstr "Gestionnaire de paquets Fortran"
+
+#: ../../pages/index.md:2
+msgid "Package manager and build system for Fortran"
+msgstr ""
+"Gestionnaire de paquets et système de construction pour le langage "
+"Fortran"
+
+#: ../../pages/index.md:19
+msgid "Welcome to the documentation for the Fortran Package Manager (fpm)."
+msgstr ""
+"Bienvenue dans la documentation du gestionnaire de paquets Fortran fpm."
+
+#: ../../pages/index.md:21
+msgid ""
+"This documentation is divided into four parts. Select one of the topics "
+"below to continue."
+msgstr ""
+"Cette documentation est divisée en quatre parties. Cliquez sur celle qui "
+"vous intéresse."
+
+#: ../../pages/index.md:1
+msgid ""
+"These pages are currently under construction. Please help us improve "
+"them by contributing content or reporting issues."
+msgstr ""
+"Ces pages sont en travaux. Aidez-nous à les améliorer en contribuant au "
+"contenu ou en rapportant des problèmes."
+
+#: ../../pages/index.md:1
+msgid "{octicon}`mortar-board` Tutorials"
+msgstr "{octicon}`mortar-board` Tutoriels"
+
+#: ../../pages/index.md:2
+msgid ""
+"Learn about using fpm for Fortran development, creating projects and "
+"managing dependencies."
+msgstr ""
+"Apprenez comment utiliser fpm pour développer en Fortran, créer des "
+"projets et gérer leurs dépendances"
+
+#: ../../pages/index.md:1
+msgid "Browse tutorials"
+msgstr "Naviguer dans les tutoriels"
```

Comment:
We can't translate the button labels yet due to a bug in sphinx-design: https://github.com/executablebooks/sphinx-design/issues/44. For the moment we should just leave them untranslated.

Reply:
Thanks Sebastian, I have just committed an updated index.po with untranslated buttons.

Reply:
I'll try contribute a fix upstream, but this requires some understanding on sphinx / docutils to figure it out.